src/compiler/diagnosticMessages.json

```diff
@@ -1011,6 +1011,10 @@
         "category": "Message",
         "code": 1350
     },
+    "An identifier cannot follow a numeric literal.": {

```

Comment:
I'm not so big on this error message. It's somewhat misleading, so the word "immediately" might help. Also, it doesn't mention keywords. What about

```
An identifier or keyword cannot follow a numeric literal.
```

I checked out Firefox which uses the word "immediately" as well:

```
SyntaxError: identifier starts immediately after numeric literal
```